google-cloud-bigtable/pom.xml

```diff
@@ -232,10 +235,12 @@
                 </goals>
                 <configuration>
                   <systemPropertyVariables>
-                    <bigtable.env>prod</bigtable.env>
+                    <bigtable.env>cloud</bigtable.env>
```

Comment:
Just thought of this - do we need to update integration.cfg to include `bigtable-directpath-it`?

Reply:
yes, but we need to whitelist the gcloud project first. This is the thing I mentioned we needed a separate kokoro pool for, but now that it's available in prod, we dont